frontend/src/Calendar/Calendar.module.css

```diff
@@ -0,0 +1,58 @@
+.calendarPage {
+    height: 100%;
+    gap: 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.calendarElem {
```

Comment:
for clarity can i suggest changing `.calendarPage` to `.container` and `.calendarElem` to `.calendar`, maybe?